packages/leavittbook/src/demos/titanium-loading-indicator/titanium-loading-indicator-playground.ts

```diff
@@ -0,0 +1,43 @@
+/* playground-fold */
+import { css, html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { h1, p } from '@leavittsoftware/titanium-styles';
+
+/* playground-fold-end */
+import '@leavittsoftware/titanium-loading-indicator/lib/titanium-loading-indicator';
+
+/* playground-fold */
+@customElement('titanium-loading-indicator-playground')
+export class TitaniumLoadingIndicatorPlayground extends LitElement {
+  static styles = [
+    h1,
+    p,
+    css`
+      :host {
+        display: flex;
+        flex-direction: column;
+        --mdc-icon-font: 'Material Icons Outlined';
+        margin: 24px 12px;
+      }
+
+      div {
+        border: 1px solid var(--app-border-color);
+        padding: 24px;
+        border-radius: 8px;
+        display: flex;
+        flex-wrap: wrap;
+        gap: 12px;
+        margin: 24px 0 36px 0;
+      }
+    `,
+  ];
+
+  render() {
+    /* playground-fold-end */
+    return html`
+      <h1>Default</h1>
+      <p>Default loading indicator</p>
+      <div><titanium-loading-indicator></titanium-loading-indicator></div>
```

Comment:
Add a disabled example

Reply:
[86293d7](https://github.com/LeavittSoftware/titanium-elements/pull/350/commits/86293d79d9e46d76d80d7335f92eddbfa9d7f46e)

Yeah, I think all of these would be useful to show. Thanks!